### DISCUSSION TITLE
What is the difference between web3.js and ethers.js?

### DISCUSSION THREAD

**Discussion by amcilwain** [community/community#30336] [Other Feature Feedback, Questions, & Ideas] [answered]
I know that both of web3.js and ethers.js are used to fetch the value on blockchain network.
But can anyone write a short comparison between the most important features of the two libraries?

**Comment by codestheos** [comment, marked as answer]
The advantage of Ethers.js is:

- smaller
- well tested
- well documented
- well maintained
- less buggy

Please check more details [here](https://ethereum.stackexchange.com/questions/62028/what-is-the-difference-between-web3-js-and-ethers-js).

**Comment by solyasa** [comment]
You can find answer in here
Both web3.js and ethers.js are JavaScript libraries that enable frontend apps to interact with the Ethereum blockchain, including smart contracts. If you're building an app that reads or writes to the blockchain from the client, you'll need to use one of these libraries. They have similar functionality, but an important question is how they will be maintained and grow with the emerging dapp ecosystem.

https://dev.to/lparvinsmith/web3js-vs-ethersjs-a-comparison-of-web3-libraries-2ap5#:~:text
Best regards
